skbio/io/format/_feature_table.py

```diff
@@ -0,0 +1,90 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2013--, scikit-bio development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+'''INSDC defined a common "feature table" specification for
+DDBJ/ENA/GenBank formats It is described
+`here<http://www.ebi.ac.uk/ena/WebFeat/>`_. The feature types and
```

Comment:
specifically I am wondering if a space character is necessary between `here` and the opening `<`